demo/app.component.ts

```diff
@@ -44,6 +44,12 @@ export class AppComponent {
       this.albums.push(album);
     }
 
+    this.albums.push({
+      src: 'demo/img/image5.pdf',
+      thumb: 'demo/img/image5-thumb.jpg',
+      iframe: true
```

Comment:
Is the iframe options needed ? I dont really understand why it is needed for an image ?